hibike/hibike_message.py

```diff
@@ -57,9 +57,11 @@
   "GenericError"        : 0xFF
 }
 
+MAX_PACKET_SIZE = 64
```

Comment:
I believe the arduino RawHID library has a 64-byte payload limit, which is where this comes from. It may be possible that HID supports bigger payloads, but that would need to be tested. And I definitely remember seeing something like `#define MAX_LENGTH 64` in the RawHID source code.